CHANGELOG.md

```diff
@@ -107,7 +109,7 @@
     `get_hint_data(self, &[HintReference], &mut dyn HintProcessor) -> Result<Vec<Box<dyn Any>, VirtualMachineError>`
   * Hook methods receive `&[Box<dyn Any>]` rather than `&HashMap<usize, Vec<Box<dyn Any>>>`
 
-#### [0.8.4] 
```

Comment:
Can we restore these unnecessary changes?

Reply:
This is done automatically by the editor. It removes any trailing spaces when saving the file.